### PR TITLE
Address issue #22

### PR DESCRIPTION
**Pull request by TroyWalshProf**
Update to support attaching result files to MSTest with .NET Core
Update for offical MAQS release 5.5.2

### CODE REVIEW

File: Framework/CoreUnitTests/Base.cs [hunk 1]
```diff
@@ -60,51 +60,6 @@ public void TestRunSettingOverrideSection()
             Assert.AreEqual("ANOTHERSAMPLE", Config.GetValueForSection("Magenicmaqs", "SectionOverrideCore"));
         }
 
-        /// <summary>
-        /// Test that in core, associated test files get written to the log
-        /// </summary>
-        [TestMethod]
-        public void TeardownWritesAssociatedFiles()
```
Comment (FermJacob) [inline Framework/CoreUnitTests/Base.cs]: Why did this test get removed?

File: Framework/BaseTest/BaseTest.cs [hunk 2]
```diff
@@ -574,7 +574,7 @@ private void AttachAssociatedFiles()
             {
                 logPath = ((FileLogger)this.Log).FilePath;
             }
-#if NET471
+
```
Comment (FermJacob) [inline Framework/BaseTest/BaseTest.cs]: Too much whitespace

File: Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt [hunk 3]
```diff
@@ -1,5 +1,11 @@
 MAQS Framework Release Notes:
-5.5.1 - 2019/8/9
+5.5.2 - 2019/9/5
+    Bug fixes and minor enhancements
+    - Add associated files to Core MSTest test results
+    Template updates
+    - Mass dependency updates
+
+5.5.1.1 - 2019/8/9
```
Comment (FermJacob) [inline Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt]: Shouldn't this be 5.5.1?

File: Framework/CoreUnitTests/Base.cs (hunk 1 above)
Comment (TroyWalshProf) [reply Framework/CoreUnitTests/Base.cs]: If we attach we don't do this associated logging

File: Framework/BaseTest/BaseTest.cs (hunk 2 above)
Comment (TroyWalshProf) [reply Framework/BaseTest/BaseTest.cs]: Are you sure, whitespace does not violate stylecop rules

File: Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt (hunk 3 above)
Comment (TroyWalshProf) [reply Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt]: Old release notes had a typo - Showed 2 5.5.1 releases
AKA - Fixed old typo

File: Framework/BaseTest/BaseTest.cs (hunk 2 above)
Comment (FermJacob) [reply Framework/BaseTest/BaseTest.cs]: You are correct. Thought there was a double return.

**Pull request merged**